src/coreComponents/mesh/mpiCommunications/CommunicationTools.cpp

```diff
@@ -686,17 +688,26 @@ void CommunicationTools::findGhosts( MeshLevel & meshLevel,
 
   auto sendGhosts = [&] ( int idx )
   {
-    neighbors[idx].prepareAndSendGhosts( false, 1, meshLevel, commID );
-    return neighbors[idx].getSizeRecvRequest( commID );
+    neighbors[idx].prepareAndSendGhosts( false,
+                                         1,
+                                         meshLevel,
+                                         commData.sizeCommID,
+                                         commData.commID,
+                                         commData.mpiRecvBufferSizeRequest[idx],
+                                         commData.mpiSendBufferSizeRequest[idx],
+                                         commData.mpiSendBufferRequest[idx] );
```

Comment:
We could pass `commData` and `idx`, but I felt it was better to be explicit about what was being passed in.